### DISCUSSION TITLE
Caribou lite installed, but dont see it on cubic sdr

### DISCUSSION THREAD

**Discussion by pierre-olivier832** [cariboulabs/cariboulite#127] [General]
Hi. I have installed the device but i dont find it on cubic sdr.
Wath am i leaving?

**Comment by meexmachina** [comment]
hi pierre, which distro are you using? cariboulite uses soapysdr as its API for cubic-sdr. in dragonos it should work fine out of the box. in Raspbian cubic-sdr is not natively installed so it may need to be installed with support for soapy.

**Comment by pierre-olivier832** [comment]
It is installed. Cariboulite with soapysdr

Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by pierre-olivier832** [comment]
I cant run dragonos, because its a raspberry pi panel pc and i cant reboot it.

Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by meexmachina** [comment]
okay got it.
can you run:
SoapySDRUtil --probe

and tell me what is the output?

**Comment by pierre-olivier832** [comment]
Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by meexmachina** [comment]
?

**Comment by pierre-olivier832** [comment]
I send you a picture.

Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by meexmachina** [comment]
pierre please note that I didn't get a picture.

**Reply by pierre-olivier832** [reply]
![20230627_192107.jpg](https://github.com/cariboulabs/cariboulite/assets/79736604/2cdc8f59-cf15-41d8-a739-7cccca9d275b)

**Comment by pierre-olivier832** [comment]
Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by meexmachina** [comment]
I think what you are experiencing is a mismatch of SoapySDR versions. The installed version comes from the aptitude (sudo apt-get install soapysdr-dev) which installs the .so files into the system and then the build was against the higher version 0.8.
What I suggest is to do the following:
remove the apt version of soapysdr
sudo apt-get remove soapysdr...
then use the most up-to-date version from git and build it.
